.test-infra/jenkins/job_PostCommit_Java_Nexmark_Flink.groovy

```diff
@@ -107,8 +115,9 @@ PhraseTriggeringPostCommitBuilder.postCommitJob('beam_PostCommit_Java_Nexmark_Fl
   commonJobProperties.setTopLevelMainJobProperties(delegate, 'master', 240)
 
   def final JOB_SPECIFIC_OPTIONS = [
-          'suite'        : 'SMOKE',
-          'streamTimeout': 60,
+          'suite' : 'SMOKE',
+          'streamTimeout' : 60,
+          'shutdownSourcesOnFinalWatermark' : true,
```

Comment:
This can also be removed if we use `TestFlinkRunner`.

Reply:
Same arguments as above. Considering the pros cons I think we have way more extra work letting things back to TestFlinkRunner, also we were supposed to remove this testrunner things no? Please remember that Nexmark is not a UT or a IT, it is a 'user application' so it should be using the real runner.